avm/ptn/aca-lza/hosting-environment/main.json

```diff
@@ -228,7 +227,7 @@
\r\n        \"frontDoorEndpoint\": \"fde\",\r\n        \"frontDoorWaf\": \"fdfp\",\r\n        \"keyVault\": \"kv\",\r\n        \"logAnalyticsWorkspace\": \"log\",\r\n        \"managedIdentity\": \"id\",\r\n        \"networkInterface\": \"nic\",\r\n        \"networkSecurityGroup\": \"nsg\",\r\n        \"privateEndpoint\": \"pep\",\r\n        \"privateLinkService\": \"pls\",\r\n        \"publicIpAddress\": \"pip\",\r\n        \"resourceGroup\": \"rg\",\r\n        \"routeTable\": \"rt\",\r\n        \"serviceBus\": \"sb\",\r\n        \"serviceBusQueue\": \"sbq\",\r\n        \"serviceBusTopic\": \"sbt\",\r\n        \"storageAccount\": \"st\",\r\n        \"virtualMachine\": \"vm\",\r\n        \"virtualNetwork\": \"vnet\",\r\n        \"redisCache\": \"redis\",\r\n        \"cognitiveAccount\": \"cog\",\r\n        \"openAiDeployment\": \"oaidep\",\r\n        \"azureAISearch\": \"srch\",\r\n        \"documentIntelligence\": \"di\",\r\n        \"eventGridSystemTopic\": \"egst\",\r\n        \"eventGridSubscription\": \"evgs\"\r\n    },\r\n\r\n    //copied from here: https://github.com/nianton/azure-naming/blob/main/datafiles/regionAbbreviations.json\r\n    \"regionAbbreviations\" : {\r\n        \"australiacentral\": \"auc\",\r\n        \"australiacentral2\": \"auc2\",\r\n        \"australiaeast\": \"aue\",\r\n        \"australiasoutheast\": \"ause\",\r\n        \"brazilsouth\": \"brs\",\r\n        \"brazilsoutheast\": \"brse\",\r\n        \"canadacentral\": \"canc\",\r\n        \"canadaeast\": \"cane\",\r\n        \"centralindia\": \"cin\",\r\n        \"centralus\": \"cus\",\r\n        \"centraluseuap\": \"cuseuap\",\r\n        \"eastasia\": \"ea\",\r\n        \"eastus\": \"eus\",\r\n        \"eastus2\": \"eus2\",\r\n        \"eastus2euap\": \"eus2euap\",\r\n        \"francecentral\": \"frc\",\r\n        \"francesouth\": \"frs\",\r\n        \"germanynorth\": \"gern\",\r\n        \"germanywestcentral\": \"gerwc\",\r\n        \"japaneast\": \"jae\",\r\n        \"japanwest\": \"jaw\",\r\n        \"jioindiacentral\": \"jioinc\",\r\n        \"jioindiawest\": \"jioinw\",\r\n        \"koreacentral\": \"koc\",\r\n        \"koreasouth\": \"kors\",\r\n        \"northcentralus\": \"ncus\",\r\n        \"northeurope\": \"neu\",\r\n        \"norwayeast\": \"nore\",\r\n        \"norwaywest\": \"norw\",\r\n        \"southafricanorth\": \"san\",\r\n        \"southafricawest\": \"saw\",\r\n        \"southcentralus\": \"scus\",\r\n        \"southeastasia\": \"sea\",\r\n        \"southindia\": \"sin\",\r\n        \"swedencentral\": \"swc\",\r\n        \"switzerlandnorth\": \"swn\",\r\n        \"switzerlandwest\": \"sww\",\r\n        \"uaecentral\": \"uaec\",\r\n        \"uaenorth\": \"uaen\",\r\n        \"uksouth\": \"uks\",\r\n        \"ukwest\": \"ukw\",\r\n        \"westcentralus\": \"wcus\",\r\n        \"westeurope\": \"weu\",\r\n        \"westindia\": \"win\",\r\n        \"westus\": \"wus\",\r\n        \"westus2\": \"wus2\",\r\n        \"westus3\": \"wus3\"\r\n    }\r\n}",
```

Comment:
is this change intended?

Reply:
I guess so. I only regenerated the json files

Reply:
Can we please test static on this one before merge?

Reply:
on it

Reply:
Attached a batch. Mind you however it will be red as the PSRule tests currently do not pass

Reply:
BTW, if I recall correctly, this is. Windows vs Linux issue, where you get two different results. Have seen this before somewhere

Reply:
And confirmed. Just re-ran it from the WSL: 
![image](https://github.com/user-attachments/assets/63d03364-8810-4ecd-888c-9181598fdef2)

These line endings will be our downfall

Reply:
Alright, passes @eriqua
I guess we'll have to look into the `settings.json` vs `extension.json` files eventually to make sense of them. Last time I tried (easily half a year ago) I failed and no configuration seemed to affect how bicep renders the line endings.